uploader.json

```diff
@@ -6943,6 +6958,38 @@
                 }
             ]
         },
+	"vdsm.conf": {
```

Comment:
See above comment

Reply:
Hey @psachin, since the `specs` part is deprecated (from last week?), you do not need to modify it in this patch.